chromium/background.js

```diff
@@ -2,6 +2,28 @@
 
 (function(exports) {
 
+var all_rules, ls;
+
+try{
+  ls = localStorage;
+} catch(e) {
+  ls = {setItem: () => {}, getItem: () => {}};
+}
+
+function initialize(){
+  // Rules are loaded here
+  all_rules = new rules.RuleSets(ls);
+
+  initializeStoredGlobals(() => {
+    all_rules.addFromJson(loadExtensionFile('rules/default.rulesets', 'json'));
```

Comment:
Could this be moved out of this callback, and into the outer function?

Reply:
If that is possible, maybe an API like this would be nice.
```javascript
// in background.initialize
all_rules = rules.initialize();
```

Reply:
It can't be moved to the outer function, since the `addFromJson` function requires us to know the stored value of `enableMixedRulesets` in https://github.com/EFForg/https-everywhere/blob/d9a4db4cc330a9f48dabb5b0f51e12918b5448c2/chromium/rules.js#L244.